### PR TITLE
Eventhub security parsing

### PR DESCRIPTION
**Pull request by verbecee**
## Description
Add initial parsing for Azure Eventhub Security Logs.

## XRefence issues
None

## Todos
N/A

### CODE REVIEW

File: config/processors/event_hub_log_security_azure.event_hub.conf [hunk 1]
```diff
@@ -6,22 +6,73 @@ input {
...
+    	source => "message"
+    	target => "tmp"
+  	}
+
+	split {
+      	field => "[tmp][records]"
+	  	target => "tcs"
+    }
+
+	mutate{
```
Comment (KrishnanandSingh) [inline config/processors/event_hub_log_security_azure.event_hub.conf:33]: A space should proceed every {
Also please double check if indentations are correct.

**Review by KrishnanandSingh — CHANGES_REQUESTED** [review]

File: config/processors/event_hub_log_security_azure.event_hub.conf (hunk 1 above)
Comment (verbecee) [reply config/processors/event_hub_log_security_azure.event_hub.conf]: Updated it, thank you!

Comment (KrishnanandSingh) [reply config/processors/event_hub_log_security_azure.event_hub.conf]: Can you make indentations with 2 spaces please?

File: config/processors/event_hub_log_security_azure.event_hub.conf [hunk 2]
```diff
@@ -15,13 +15,64 @@ filter {
...
+  json {
+    source => "message"
+    target => "tmp"
+  }
+
+  split {
+    field => "[tmp][records]"
+    target => "tcs"
+  }
+
```
Comment (KrishnanandSingh) [inline config/processors/event_hub_log_security_azure.event_hub.conf:32]: Original message field has list of events. After the split, each event would have original message as list of all events which is not needed. After the split please add

```suggestion
 json_encode {
    source => "tcs"
    target => "message"
  }
```

We have done the same in azure audit and okta.
@brian-grabau Do you agree?

**Review by KrishnanandSingh — CHANGES_REQUESTED** [review]

File: config/processors/event_hub_log_security_azure.event_hub.conf (hunk 2 above)
Comment (brian-grabau) [reply config/processors/event_hub_log_security_azure.event_hub.conf:32]: Yes that is probably true, split would split into separate messages

**Review by KrishnanandSingh — APPROVED** [review]

**Pull request merged**